tests/roots/test-minimal/conf.py

```diff
@@ -0,0 +1,3 @@
+# minimal test root
+include_patterns = ['index.rst']
+exclude_patterns = ['_build']
```

Comment:
Are these two configuration options required?

Reply:
I'm not sure that the exclude_patterns include the '_build' by default. I think the default exclude patterns are `[]`. It's just to speed-things up when finding iterating the files.

Reply:
Ok, thanks.  This is really a question in case anyone later thinks "ah, this is the absolute minimal Sphinx project" (based on the dirname).  A `conf.py` file does seem to be required for Sphinx to build a project, but it can be empty.

If we're not outputting files into this test-root during the tests (and I don't think we should be?), then we could remove the `exclude` setting at least.  Probably the `include_patterns` too.